aws/resource_aws_spot_instance_request.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/hashicorp/terraform/helper/schema"
 )
 
+const awsSpotInstanceRequestTimeLayout = "2006-01-02T15:04:05Z"
```

Comment:
Instead of creating our own time layout, can we please use `time.RFC3339` instead?